io.openems.backend.metadata.ldap/bnd.bnd

```diff
@@ -0,0 +1,16 @@
+Bundle-Name: OpenEMS Backend Metadata LDAP
+Bundle-Version: 1.0.0.${tstamp}
+Bundle-Description: Allows authentication, authorization and edge configuration through a LDAP server.
+Bundle-Category: osgi
+Bundle-Vendor: Christian Poulter
+Bundle-ContactAddress: devel(at)poulter.de
+Bundle-Copyright: OpenEMS Backend Metadata LDAP (C) 2024 Christian Poulter
+Bundle-License: https://www.gnu.org/licenses/agpl-3.0.txt
```

Comment:
Mixing AGPL License and EPL License codes in one binary is very difficult to use.
Because AGPL infect to other files, and it require redistribution these files by AGPL License.
This means existing EPL license code need to change AGPL license.
OpenEMS is OSS software and authors are many. So it is difficult to get permission to change license from ALL original content right holders(authors).

I think better to change license to EPL, if you can agree it.

Reply:
// I feel this PR is very very nice! and exciting. and I don't hate AGPL/EPL License.
// I love OSS license and culture, so I just pointed out the license problem.

Reply:
Hi @miettal ... thanks for your feedback. I am not so familiar with all these OSS licenses. At the moment reading EPL in opposite to GPL feels a bit like "you can contribute for free, but my modules stay closed". I will think about it and might change the PR.

Reply:
I found useful EPL official FAQ. I share it. This kind matter is complex and not easy to understand... I'm also studying now.

https://www.eclipse.org/legal/epl-2.0/faq.php#h.5ucozq4kvv7o

> [4.14. Does the EPL allow me to take the Source Code for a Program licensed under it and include all or part of it in another program licensed under the GNU General Public License (GPL), Berkeley Software Distribution (BSD) license or other Open Source license?](https://www.eclipse.org/legal/epl-2.0/faq.php#h.5ucozq4kvv7o)
No. Only the owner of software can decide whether and how to license it to others. Contributors to a Program licensed under the EPL understand that source code for the Program will be made available under the terms of the EPL. Unless you are the owner of the software or have received permission from the owner, you are not authorized to apply the terms of another license to the Program by including it in a program licensed under another Open Source license.

> However, if the content has been made available under the EPL-2.0 with Secondary License, the content may be distributed under the terms of that Secondary License while maintaining the original code base under the EPL-2.0.

> [4.15. Is my EPL-2.0 licensed project GPL compatible by default?](https://www.eclipse.org/legal/epl-2.0/faq.php#h.fn86eyf5mvhs)
No. EPL-2.0 licensed projects may explicitly add GPL compatibility by way of adding a Secondary License as per Exhibit A — Form of Secondary Licenses Notice. See Section 3.2 of the EPL-2.0. Please note adding a secondary license is considered a relicensing exercise and all copyright holders must agree to the license change.

note: This PR and code is your content. Deciding a license for this code is your right. Anyone cannot force it.